test-cloud/test-cloud-remote/pom.xml

```diff
@@ -72,6 +72,12 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
     </dependency>
+    
+    <dependency>
+      <groupId>org.awaitility</groupId>
+      <artifactId>awaitility</artifactId>
+      <scope>compile</scope>
```

Comment:
just minor thing, do we need there scope?

Reply:
Yes, because the kie-parent is marked this dependency with the scope "test", and we created an utility wrapper in the sources.